xbmc/LangInfo.cpp

```diff
@@ -1133,6 +1134,11 @@ void CLangInfo::SettingOptionsStreamLanguagesFiller(const CSetting *setting, std
 
   std::string dummy;
   SettingOptionsISO6391LanguagesFiller(NULL, list, dummy, NULL);
+  SettingOptionsLanguageNamesFiller(NULL, list, dummy, NULL);
+  std::set<std::pair<std::string, std::string>> languages;
+  unsigned size = list.size();
+  for (unsigned i = 0; i < size; ++i) languages.insert(list[i]);
```

Comment:
I would use the set contructor or `std::set::insert()`. I don't think that the speed impact will be noticeable at all because the list is not that long and it's much easier to read.